src/Helper/DrupalApiHelper.php

```diff
@@ -83,6 +102,24 @@ public function getBundles()
     /**
      * @return array
      */
+    public function getRoles()
+    {
+        if (!$this->roles) {
+            $roles = Role::loadMultiple();
```

Comment:
we had the acces to the entity manager service 
`$entityManager = $this->hasGetService('entity.manager');`
and this be used insted of static method calls `Role::loadMultiple();`